src/tlo/methods/healthburden.py

```diff
@@ -99,6 +101,7 @@ def initialise_simulation(self, sim):
         self.years_life_lost_stacked_time = pd.DataFrame(index=self.multi_index_for_age_and_wealth_and_time)
         self.years_life_lost_stacked_age_and_time = pd.DataFrame(index=self.multi_index_for_age_and_wealth_and_time)
         self.years_lived_with_disability = pd.DataFrame(index=self.multi_index_for_age_and_wealth_and_time)
+        self.prevalence_of_diseases = pd.DataFrame(index=year_index)
```

Comment:
if we want it to take different frequencies (e.g month, year etc) then we'd need a difference index.